projects/ngx-device-detector/src/lib/device-detector.service.ts

```diff
@@ -189,7 +189,7 @@ export class DeviceDetectorService {
    * @returns whether the current device is a desktop device
    */
   public isDesktop(userAgent = this.userAgent): boolean {
-    const desktopDevices = [Constants.DEVICES.PS4, Constants.DEVICES.CHROME_BOOK, Constants.DEVICES.UNKNOWN];
+    const desktopDevices = [Constants.DEVICES.PS4, Constants.DEVICES.CHROME_BOOK, Constants.DEVICES.MACINTOSH, Constants.DEVICES.UNKNOWN];
```

Comment:
This seems like a hack we're adding here. Do we need to do this? How was this handled previously?